api/app/docker-compose.dev.yml

```diff
@@ -33,6 +33,8 @@ services:
       - "9229:9229"
     volumes:
       - ./src:/usr/src/app/src
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
Allows us to use `host.docker.internal` to access the host's IP and, thus, other apps running local